docs/user/azure/limits.md

```diff
@@ -0,0 +1,124 @@
+# Limits
+
+You can find a comprehensive list of the default Azure service limits published here:
+
+[Azure Service Limits][service-limits]
+
+Below, we'll identify OpenShift cluster needs and how those impact some of those limits.
+
+## VNet
+
+Each cluster creates its own VNet. The default limit of VNets per regions is 1000 and will allow 1000 clusters. To 
+have more than 1000 clusters, you will need to increase this limit. The default installation creates a single VNet.
+
+## Network Interfaces
+
+The default installation creates 6 network interfaces. The default limit per region is 65536. Additional network interfaces are
+created for additional machines and load balancers created by cluster usage and deployed workloads.
+
+## Public and Private IP Addresses
```

Comment:
this section doesn't provide the number of public addressed used by the cluster. one for external lb, one for bootstrap-host (which is removed after installation is complete)

Reply:
Updated.